flowmachine/flowmachine/core/server/query_schemas/per_subscriber_aggregate.py

```diff
@@ -0,0 +1,46 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+from functools import reduce
+
+from marshmallow import fields
+from marshmallow.validate import OneOf, Length
+
+from flowmachine.core.server.query_schemas import BaseExposedQuery
+from flowmachine.core.server.query_schemas.numeric_subscriber_metrics import (
+    NumericSubscriberMetricsSchema,
+)
+from flowmachine.core.server.query_schemas.base_schema import BaseSchema
+from flowmachine.features.subscriber.per_subscriber_aggregate import (
+    PerSubscriberAggregate,
+    agg_methods,
+)
+
+
+class PerSubscriberAggregateExposed(BaseExposedQuery):
+    def __init__(self, subscriber_queries, agg_method):
+        self.subscriber_queries = subscriber_queries
+        self.agg_method = agg_method
+
+    @property
+    def _flowmachine_query_obj(self):
+        subscriber_query = reduce(
+            # TODO: Replace with Jono's new list input to union
+            lambda x, y: x._flowmachine_query_obj.union(y._flowmachine_query_obj),
+            self.subscriber_queries,
```

Comment:
Actually, this won't work - sorry. You want `reduce(lambda x, y: x.union(y), (q._flowmachine_query_obj for q in self.subscriber_queries)`

Reply:
It's going to be moot soon - Jono's working on #4593 , we may as well wait for that and incorporate it into this PR.